anitab-org/bridge-in-tech-web
# Feat: Issue#27 view user personal details

### Description
Add feature: View user's personal details

Fixes #27

### Type of Change:
- Code
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
- login as user
- go to My Space -> My Profile page
- See Personal Details
![ezgif com-video-to-gif (1)](https://user-images.githubusercontent.com/29667122/87239942-0c416580-c458-11ea-9d43-f3e1b9977182.gif)


### Checklist:



**Code/Quality Assurance Only**
